tests/st/utils/data.py

```diff
@@ -375,9 +363,13 @@
     'apiVersion': API_VERSION,
     'kind': 'WorkloadEndpoint',
     'metadata': {
+        'labels': {
```

Comment:
namespace should be namespace1 according to the spec

Reply:
will fix

Reply:
well it may not be necessary - since the test could actually check that the values are updated to their correct values... not sure.

Reply:
actually scrap that - i think both of these WEP definitions need to be valid because there are several tests that use these defs - so unless you wanna fix up a bunch of tests it's prolly easier to have these valid.